cmake/packaging/linux.cmake

```diff
@@ -79,7 +79,8 @@ if(${SUNSHINE_TRAY} STREQUAL 1)
 
     set(CPACK_DEBIAN_PACKAGE_DEPENDS "\
                     ${CPACK_DEBIAN_PACKAGE_DEPENDS}, \
-                    libappindicator3-1")
+                    libayatana-appindicator3-1, \
+                    libnotify4")
     set(CPACK_RPM_PACKAGE_REQUIRES "\
                     ${CPACK_RPM_PACKAGE_REQUIRES}, \
                     libappindicator-gtk3 >= 12.10.0")
```

Comment:
Do we need to fix this one too?

Reply:
Probably... https://packages.fedoraproject.org/pkgs/libnotify/libnotify/

Reply:
Using a vanilla Fedora 38 docker image, seems fine to me - no missing dependencies.

```
yum check-update
yum localinstall /sunshine.rpm
sunshine
```

I'm not exactly sure what in the tree is installing libnotify.so.4 but it's there after the `localinstall`.

Maybe in a future effort, the GitHub actions can start containers for the images they built, install the Sunshine package, and run a quick `sunshine --version` just to make sure that all the dependencies are in order.

Reply:
It's probably included in Fedora distros, but maybe we should add it for more minimal distros.

The sunshine package is actually installed in the docker images... but we don't run it right now. As far as I understand this would add an additional layer to the docker image which would not be desirable (although I don't think it would necessarily be the end of the world)

https://github.com/LizardByte/Sunshine/blob/56d13bc7e84db160dfbea1cbf94fd1722f5e8b32/docker/fedora-38.dockerfile#L135-L142

Reply:
According to `repoquery --requires /sunshine.rpm --tree`, libnotify is a top level dependency, just not sure where it's getting added in the CMake...

As for the Docker image, I agree we shouldn't do this test in the Dockerfile itself. My suggestion was in the GitHub Actions, after the Docker image is built, spin up a container using that image and make sure Sunshine can be executed. Not sure if this is possible as I'm not too familiar with GitHub actions.

Reply:
> just not sure where it's getting added in the CMake

My guess is `libappindicator-gtk3`, not 100% sure though. I can't see anyway to find dependencies on the fedora package site. This seems logical though as we kept this legacy appindicator on fedora builds.

Reply:
I will merge this and we can deal with rpm later if it becomes an issue. Thank you!